medical/models/medical_patient.py

```diff
@@ -71,9 +75,13 @@ def _compute_age(self):
                     delta.years, _('y'), delta.months, _('m'),
                     delta.days, _('d'), is_deceased
                 )
+                years = delta.years
             else:
                 years_months_days = _('No DoB')
+                years = False
             record.age = years_months_days
+            if years:
```

Comment:
This conditional doesn't seem necessary since setting the value of the field to `False` should work just fine and is equivalent to leaving it unset.